package.json

```diff
@@ -41,20 +41,19 @@
     "sprintf-js": "^1.1.1"
   },
   "scripts": {
-    "check-gb-unused-babelrc": "cd gutenberg/; git ls-files --error-unmatch .babelrc 2>/dev/null; tracked=$?; cd ..; (if [ \"$tracked\" -gt 0 ]; then (exit 0); else (echo \"'.babelrc' is tracked in Gutenberg, cannot overwrite!\"; exit 1); fi)",
-    "bypass-gb-babel": "echo '{}' > gutenberg/.babelrc",
-    "test:bypass-gb-babel": "echo '{\"presets\": [\"@wordpress/default\"]}' > gutenberg/.babelrc",
-    "pre-build": "yarn check-gb-unused-babelrc && yarn bypass-gb-babel",
-    "test:pre-build": "yarn check-gb-unused-babelrc && yarn test:bypass-gb-babel",
-    "start": "yarn pre-build && yarn react-native start",
+    "start": "yarn react-native start",
+    "start:inside-gb": "yarn react-native start --config `pwd`/rn-cli-inside-gb.config.js",
     "start:reset": "yarn clean:runtime && yarn start --reset-cache",
-    "start:debug": "yarn pre-build && node --inspect-brk node_modules/.bin/react-native start",
+    "start:inside-gb:reset": "yarn clean:runtime && yarn start:inside-gb --reset-cache",
+    "start:debug": "node --inspect-brk node_modules/.bin/react-native start",
```

Comment:
This doens't look right, shouldn't this be kept has before? At the moment this command looks exactly the same has the one bellow.